sdk/spring/azure-spring-boot-starter-active-directory-b2c/pom.xml

```diff
@@ -98,7 +98,7 @@
           <rules>
             <bannedDependencies>
               <includes>
-                <include>com.microsoft.azure:azure-spring-boot:[2.4.0-beta.1]</include> <!-- {x-include-update;com.microsoft.azure:azure-spring-boot;current} -->
+                <include>com.azure:azure-spring-boot:[3.0.0-beta.1]</include> <!-- {x-include-update;com.azure:azure-spring-boot;current} -->
```

Comment:
We don't need this anymore, the parent pom already adds exclusions for com.azure:*.